akka-sample-fsm-java/build.sbt

```diff
@@ -1,14 +1,14 @@
 organization := "com.typesafe.akka.samples"
 name := "akka-sample-fsm-java"
 
-val akkaVersion = "2.6.0-M1"
+val akkaVersion = "2.6.0-RC1"
 
 scalaVersion := "2.12.8"
 libraryDependencies ++= Seq(
-  "com.typesafe.akka" %%       "akka-actor" % akkaVersion,
   "com.typesafe.akka" %% "akka-actor-typed" % akkaVersion,
+  "ch.qos.logback" % "logback-classic" % "1.2.3",
```

Comment:
also add logback.xml configuration, see for example https://github.com/akka/akka-samples/pull/135